test/autodiff.in.cpp

```diff
@@ -1364,7 +1364,7 @@ NGRAPH_TEST(${BACKEND_NAME}, backwards_tan)
     auto pi = 3.14159f;
 
     // Stay away from the asymptotes at 6 and 12 o'clock.
-    auto slop = 0.1f;
+    auto slop = 0.2f;
```

Comment:
To stay further away from 6 and 12 o'clock

Reply:
No objection from me... `0.1f` was kind of arbitrary in the first place.